torch/csrc/generic/methods/TensorMath.cwrap

```diff
@@ -757,6 +757,36 @@
       return: accreal
       arguments:
         - THTensor* self
+        - arg: bool unbiased
+          negated: True
+    - cname: varall
+      return: accreal
+      arguments:
+        - THTensor* self
+        - CONSTANT false
```

Comment:
It's just PyYAML that parses `true` as a Python boolean. `1` or `"true"` would both work and I'm fine with both